client/src/components/CreateAQuestion/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { createRef, useRef } from 'react';
+// import { useGameContext } from '../../utils/GlobalState';
+import { ADD_QUESTION } from '../../utils/actions';
+import Button from "../Button";
+import Toggle from "../Toggle";
+
+const CreateAQuestion = () => {
+    const topicRef = useRef();
+    const questionRef = useRef();
+    const questionTypeRef = createRef();
+    const answersRef = createRef();
+    const correctRef = createRef();
+    
+    const handleSubmit = event => {
+        alert("Question Saved!")
+        
+        dispatch({
```

Comment:
@dropcat13 dispatch is used to add the data to the `globalstate` - I think what you want to do here is pass it to `API.createQuestion` (which doesn't exist yet :wink:).